packages/elements/src/clock/__test__/clock.test.js

```diff
@@ -25,7 +25,7 @@ describe('clock/Clock', () => {
       el = await fixture('<ef-clock analogue></ef-clock>');
       el.style.width = '129px';
       await elementUpdated(el);
-      await nextFrame();
+      await nextFrame(2); // Chrome 111 & Firefox 111 needs another frame to complete rendering
```

Comment:
I can confirm the native resize observer makes some test cases need one extra frame. Please update the code comments.

Reply:
comment updated.